### DISCUSSION TITLE
Allowing to initiate Sepa Payment by scanning QR Code

### DISCUSSION THREAD

**Discussion by hoschi-it** [bisq-network/bisq#7001] [Ideas]
For Sepa Payments there is a [QR code standard](https://en.wikipedia.org/wiki/EPC_QR_code) for paying invoices.

In my opinion, paying from mobile would be much easier and faster, when the payment details screen would additionally show a QR code to scan.
This feature could also be hidden by a simple button.

What do you think?

**Comment by pazza83** [comment]
Hi @hoschi-it thanks for the question. 

Sounds like a good idea. Is this used by many banks yet?

I imagine a new QR code would be needed per trade (different amounts needed etc)

@jmacxx is this something that would be possible?

**Reply by ghost** [reply]
Yes, I am working on adding QR payment codes for SEPA/SEPA Instant.

Spec: https://en.wikipedia.org/wiki/EPC_QR_code

**Reply by ghost** [reply]
Implementation PR - https://github.com/bisq-network/bisq/pull/7005

**Comment by hoschi-it** [comment]
>Is this used by many banks yet?

I don't have any insights on that, but my bank supports it and about half of the invoices I pay make use of it.

Sure, as the amount to send is part of the standard, there should be a different QR code for every trade. (Generating it on the fly?)
